eventV1.go

```diff
@@ -21,6 +26,9 @@ type eventV1 struct {
 	EventIDRaw string           `json:"event_id,omitempty"`
 	PrevEvents []eventReference `json:"prev_events"`
 	AuthEvents []eventReference `json:"auth_events"`
+
+	UnstableSticky stickyEventData `json:"msc4354_sticky,omitempty"`
+	StableSticky   stickyEventData `json:"sticky,omitempty"`
```

Comment:
Should this be applied to `eventV1`?
At a glance, I would think this only applies to `eventV3` or newer. But I haven't been following gmsl or that MSC so maybe I'm wrong.